components/Carousel.js

```diff
@@ -34,33 +33,37 @@ const EmptyDiamond = styled.div`
     transform: rotateY(0deg) rotate(45deg);
     margin-right: 5px;
     margin-left: 5px;
+    cursor: pointer;
 `;
 
 const Image = styled.img`
-    width: ${(props) => props.width};
-    height: ${(props) => props.height};
+    width: 29vw;
+    height: 25hw;
     margin-bottom: 10px;
     border-radius: 6px;
     object-fit: cover;
+    ${(p) => p.theme.mediaQueries.mobile} {
+        width: 100%;
+        height: 100%;
```

Comment:
Can we set a static height like we did for the desktop view? On mobile the different heights between images leads to a "jumpy" experience whenever the picture changes!